docs/ref/databases.txt

```diff
@@ -367,28 +367,6 @@ this for you (the database backend layer and the model population layer are
 separated internally so the database layer doesn't know it needs to make this
 conversion in this one particular case).
 
-If you're using MySQLdb 1.2.1p2, Django's standard
```

Comment:
@timgraham Afraid I didn't know about this case, I have used binary text columns (`ut8mb4_bin` 😉  ) with `mysqlclient` in the past and only ever had Python 3 `str`s afaicr.